packages/supabase/README.md

```
@@ -9,19 +21,307 @@ npm install @tuturuuu/supabase
+
+## Best Practices
+
+1. **Client vs Server**: Always use the appropriate client creation function:
+   - Browser: `createClient()` from `/next/client`
+   - Server: `createClient()` from `/next/server`
+   - Admin: `createAdminClient()` from `/next/server`
+
+2. **Security**: Never expose the service role key in client-side code. Use `createAdminClient()` only in server-side contexts.
+
+3. **Error Handling**: Always check for errors when making Supabase queries:
+   ```typescript
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)